rmgpy/rmg/input.py

```diff
@@ -225,6 +225,19 @@ def simple_reactor(temperature,
     if not isinstance(temperature, list) and not isinstance(pressure, list) and all(
             [not isinstance(x, list) for x in initialMoleFractions.values()]):
         nSims = 1
+        # normalize mole fractions if applicable
+        total_initial_moles = sum(initialMoleFractions.values())
+        if total_initial_moles != 1:
+            logging.warning('Initial mole fractions do not sum to one; normalizing.')
+            logging.info('Original composition:')
+            for spec, molfrac in initialMoleFractions.items():
+                logging.info('{0} = {1}'.format(spec, molfrac))
+            for spec in initialMoleFractions:
+                initialMoleFractions[spec] /= total_initial_moles
+            logging.info('Normalized mole fractions:')
+            for spec, molfrac in initialMoleFractions.items():
+                logging.info('{0} = {1}'.format(spec, molfrac))
+            logging.info('')
```

Comment:
This is really minor, but I think it would be nice to put the blank lines before `Original composition` and `Normalized mole fractions` like [before](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/5a07cc9aa932c0d2d031d81254381caa48f9bd83/rmgpy/rmg/input.py#L151).